dot/parachain/candidate-validation/host.go

```diff
@@ -0,0 +1,80 @@
+package candidatevalidation
+
+import (
+	"fmt"
+
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+var logger = log.NewFromGlobal(log.AddContext("pkg", "pvf"), log.SetLevel(log.Debug))
```

Comment:
You are in package `candidatevalidation`, but you are using a logger with `pkg` as `pvf`

Reply:
Updated.